src/App.js

```diff
@@ -1,36 +1,60 @@
-import React from "react";
-import TodoList from "./components/TodoComponents/TodoList.js";
-import TodoForm from "./components/TodoComponents/TodoForm.js";
+import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 import './components/TodoComponents/Todo.css';
 
 class App extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      todoData: []
-    };
-  }
-
-  addTask = task => {
-    const newTask = {
-      task: task,
-      id: Date.now(),
-      isCompleted: false
-    };
-
-    this.setState({
-      todoData: [...this.state.todoData, newTask]
-    });
-
-    console.log(this.state.todoData);
-  };
-
+    constructor() {
+      super();
+      this.state = {
+        tasks: [],
+        holder: "",
+      }
+    }
+    toggleClass = (event, id) => {
+      event.target.classList.toggle('completed');
+      this.setState({
```

Comment:
Overall, your project is looking good Rob. Take a look at the snippet below and the comments 
```javascript
let tasks = this.state.tasks.slice(); // copy of todos

    tasks = tasks.map(task => {
      if (task.id === id) {
        task.completed = !task.completed; // only update the prop that has been changed
        return task; // return the updated task
      } else {
        return task;
      }
    });

    this.setState({ tasks }); // setting state w/new tasks array created above instead of from within the setState method. 
```
We should try and avoid placing too much logic inside `setState` itself. Check out this [Article on setState](https://itnext.io/react-setstate-usage-and-gotchas-ac10b4e03d60)